addons/mail/static/src/chatter/web_portal/chatter.js

```diff
@@ -328,7 +327,11 @@ export class Chatter extends Component {
         this.toggleComposer();
         this.state.jumpThreadPresent++;
         // Load new messages to fetch potential new messages from other users (useful due to lack of auto-sync in chatter).
-        this.load(this.state.thread, ["followers", "messages", "suggestedRecipients"]);
+        this.loadAfterPost();
```

Comment:
Shouldn't this simply use `requestList` + messages rather than be separate method that is overriden?

Reply:
Ok, but then at least `loadAfterPost` should just return the params (similar than requestList) rather than duplicating the call to the load method.